src/main/java/ch/liquidmind/inflection/proxy/ProxyGenerator.java

```diff
@@ -40,6 +40,10 @@ public static void main( String[] args )
 		for ( String taxonomyName : taxonomyNames )
 		{
 			Taxonomy taxonomy = TaxonomyLoader.getContextTaxonomyLoader().loadTaxonomy( taxonomyName );
+			if (taxonomy == null) {
```

Comment:
I have replaced the change in ProxyGenerator with a TODO, so we can find a solution later